bulletin/index.html

```diff
@@ -177,9 +177,55 @@ <h3 class="buarticle-scholarships--title">MACILE Scholarships Program </h3>
 				<img src="/images/bulletin/scholarships/andreina_troncoso.jpg">
 				<p>Andreina De Jesús T., 9th grade, Colegio Padre Zegri, Nigua.</p>
 			</article>
-			
 		</div>	
-	</div>
+	</div> 
+	<section class="feature-article">
+		<div class="feature-article--wrapper">
+			<h2 class="feature-article--section-heading">Featured Article</h2>
+			<div class="article">
+				<div class="article-banner">
+					<h1 class="article-banner--brand" itemprop="brand">MACILE</h1> 
+					<!-- <h3 class="article-banner--slogan">Advancing Engineering and Technology Education</h3> -->
```

Comment:
The h1 headers are not correct HTML. You can't have an `h2` before and `h1`. `section` allows you to reset all headers. You could have an `h1` and then an `h2`.

Also, I don't understand why this section needs to repeat the organization name again. 

Please remove L188. All code that is commented out should always be removed.

Reply:
Why aren't h1 headers correct HTML?

I agree it is not correct to use h1 after h2. In my design plan this should be h2. No sure why I made the mistake.

True, h1 here is a placeholder for the MACILE logo; an image.

Reply:
My bad, that was not worded well. What I meant to say is that it isn't correct because it appears before the `h2`. Since it is a placeholder, can you use a `span`?

Reply:
Tried using span, but did not work well. Used h2 instead.

Reply:
You can't use an h2. it isn't semantic. Please see my additional commits. I fixed the issue.